apps/frontend/src/components/ThisWeek/ThisWeekComponent.tsx

```diff
@@ -0,0 +1,120 @@
+"use client"
```

Comment:
Why using this here? I don't see any user-interactions added as of now other than logging.